metapool/count.py

```diff
@@ -81,26 +83,45 @@ def _parsefier(run_dir, sample_sheet, subdir, suffix, name, funk):
     """
     out = []
 
-    for project, lane in {(s.Sample_Project, s.Lane) for s in sample_sheet}:
+    is_sample_sheet = True if isinstance(metadata,
+                                         metapool.KLSampleSheet) else False
+
+    if is_sample_sheet:
```

Comment:
Could you merge these lines to:
if isinstance(metadata, metapool.KLSampleSheet):